src/main/scala/com/snowflake/snowpark/internal/OpenTelemetry.scala

```diff
@@ -12,21 +12,17 @@ object OpenTelemetry extends Logging {
   val spanInfo = new DynamicVariable[Option[SpanInfo]](None)
 
   // wrapper of all action functions
-  def action[T](className: String, funcName: String, isScala: Boolean)(func: => T): T = {
+  def action[T](className: String, funcName: String, isScala: Boolean, javaOffSet: Int = 0)(
```

Comment:
introduced `javaOffset` parameter. In some cases, Java API wrapped more than one layers on Scala API.